Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -383,12 +383,17 @@ public function findBlockEnd($type, $searchIndex, $findEnd = true)
      * Find tokens of given kind.
      *
      * @param int|array $possibleKind kind or array of kind
+     * @param int|null  $start        optional offset
```

Comment:
tbh first time I red this i though it;s null too, just for the convention you mentioned